docs/products/flink/howto/create-sql-application.rst

```diff
@@ -0,0 +1,66 @@
+Create an SQL application
+============================
+
+In Aiven for Apache Flink®, you can create an SQL application that uses Apache Flink SQL to streamline the process of building data processing pipelines. The SQL application simplifies the process of defining source and sink tables, implementing data processing logic, and managing application deployment.
+
+Prerequisite
+------------
+Before creating applications, configure the :doc:`data service integration </docs/products/flink/howto/create-integration>` for seamless integration and data management within your Flink applications.
+
+Create and deploy application
+-------------------------------
+
+Create an SQL application in Aiven for Apache Flink® using the `Aiven Console <https://console.aiven.io/>`_:
+
+1. In the `Aiven Console <https://console.aiven.io/>`_, select the Aiven for Apache Flink service where you want to create and deploy a Flink application.
+2. From the left sidebar, click **Applications** and then click **Create new application**.
+3. In the **Create new application** dialog, enter the name of your application and select **SQL** as the application type. 
+4. Click **Create application**.
+5. Click **Create first version** to create the first version of the application. 
+6. Click **Add your first source table** to add a source table. 
+   
+   .. note::
+      As this is your first application, no other applications are available to import source tables.
```

Comment:
In the spirit of value and word economy, do we need this note? Are we anticipating user questions for this? (same question for the next note too)

Reply:
I would yes for now. It addresses a potential point of confusion for new users who might see the import table option in their first application despite having no other applications to import from. In an ideal scenario, the import option would be disabled or not visible for a user's first application to prevent this confusion. 

While we haven't received reports of any issues, we want to preemptively clarify any potential questions or confusion by including this note.